### PR TITLE
Feature/eosc client

### PR DESCRIPTION
**Pull request by sndvsk**
@Fronox

### CODE REVIEW

File: src/client_waldur.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+# Waldur Organization -> http://localhost/api/customers/
+# Waldur Project      -> http://localhost/api/projects/
+# Waldur Order        -> http://localhost/api/marketplace-offerings/
+# Add new members     -> http://localhost/api/user-invitations/
+#   Cancel invitation -> http://localhost/api/user-invitations/<invitation_id>/cancel/
+
+# Remove organization is not working
+# http://localhost/api/support-issues/
+# Status Code: 424 Failed Dependency
+# {"detail":"Extension is disabled."}
```
Comment (sergei-zaiaev) [inline src/client_waldur.py]: Please add a new line

File: src/app.py [hunk 2]
```diff
@@ -1,11 +1,29 @@
...
+def to_file(filename, json_content):
+    with open(filename, 'w', encoding='utf-8') as f:  # data to json file
+        json.dump(json_content, f, ensure_ascii=False, indent=4)  # file with nice markdown
+
+
+res = get_events_from_eosc()
+to_file('data.json', res)
+# file is created in repository only after stopping the application
+
+# TODO: print result to console
```
Comment (sergei-zaiaev) [inline src/app.py]: Why to keep this TODO?

File: .gitignore [hunk 3]
```diff
@@ -5,6 +5,7 @@
 .cache/
 .pytest_cache/
 src/test_files
```
Comment (sergei-zaiaev) [inline .gitignore]: Basically, you can just not add these files to the git repository, so these two lines can be dropped. For instance, don't run `git add src/test_files` and `git add src/data.json` or `git add .`

File: src/test_files/test.py [hunk 4]
```diff
@@ -27,15 +27,12 @@
...
-packages_str = json.dumps(packages_json)
-
-# print(response.status_code)
-# print(packages_json)
-# print(test_time)
 
 
-with open('data.json', 'w', encoding='utf-8') as f:  # data to json file
-    json.dump(packages_json, f, ensure_ascii=False, indent=4)   # file with nice markdown
+def to_file(filename):
```
Comment (sergei-zaiaev) [inline src/test_files/test.py]: Do we still need this function?

File: src/client_eosc.py [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+    response = requests.get(EOSC_URL,
+                            headers=headers,
+                            params=params,
+                            verify=False)  # because of _ssl.c:1124 [SSL: CERTIFICATE_VERIFY_FAILED]
+
+    # noinspection PyBroadException
+    # TODO: error handling (try/except block); by default, return [] or None
+    try:
+        return response.json()
+
```
Comment (sergei-zaiaev) [inline src/client_eosc.py:32]: Drop this blank line please

File: src/app.py [hunk 6]
```diff
@@ -1,11 +1,29 @@
...
+print("Output:\n", res)
```
Comment (sergei-zaiaev) [inline src/app.py]: It can be used for testing, no need for it in the repository

**Review by sergei-zaiaev — COMMENTED** [review]
After these questions resolving, drop the "draft" status. And resolve conflicts with the main branch, please.

File: src/client_waldur.py (hunk 1 above)
Comment (sndvsk) [reply src/client_waldur.py]: done

File: src/app.py (hunk 2 above)
Comment (sndvsk) [reply src/app.py]: deleted, done

File: .gitignore (hunk 3 above)
Comment (sndvsk) [reply .gitignore]: deleted, done

File: src/test_files/test.py (hunk 4 above)
Comment (sndvsk) [reply src/test_files/test.py]: I also want to test client_waldur.py in this file, so I want to keep this function as by now.

File: src/client_eosc.py (hunk 5 above)
Comment (sndvsk) [reply src/client_eosc.py:32]: deleted singular space, that was there.
kept the line, because of file formatting

File: src/app.py (hunk 6 above)
Comment (sndvsk) [reply src/app.py]: deleted, done

**Pull request merged**